docs/cli-deploy.md

```diff
@@ -68,8 +68,10 @@ pnpm sphinx deploy <SCRIPT_PATH> --network <NETWORK_NAME> [options]
 
 ### Options
 - `--network <NETWORK_NAME>`: **Required**. The name of the network to deploy on, which must match a network in the `rpc_endpoints` section of your `foundry.toml`.
+- `--sig <SIGNATURE [PARAMETERS...] | CALLDATA>` (Alias: `-s`): **Optional**. The signature of the function to call in the script, or raw calldata. Matches the interface of Forge Script's `--sig` parameter.
```

Comment:
The alias listed here is `-s`, but the alias for `--target-contract` is `--tc` which has two dashes. I would expect these to be the same format. Is one of them wrong?

Reply:
Nope, they're actually both correct. This matches Foundry's aliases, which have a different number of dashes for whatever reason. Even though it's bizarre, I think we should maintain the same UX as Foundry to make existing script commands portable to Sphinx

Reply:
To be clear, `--s` and `-tc` both don't work for Forge Scripts